public/components/models/models.controller.js

```diff
@@ -346,80 +309,43 @@
+                    $scope.status = modelService.getStatus($scope.models);
 
-                        refreshModels();
-                    })
-                    .catch(function (err) {
-                        loggerService.error('[ml4kmodels] model training failed', err);
-                        $scope.submittingTrainingRequest = false;
-
-                        if (createModelFailedDueToDownloadFail(err)) {
-                            return $mdDialog.show({
-                                controller : function ($scope) {
-                                    $scope.location = err.data.location;
-
-                                    $scope.hide = function() {
-                                        $mdDialog.hide();
-                                    };
-                                    $scope.cancel = function() {
-                                        $mdDialog.cancel();
-                                    };
-                                    $scope.confirm = function() {
-                                        $mdDialog.hide(err.data.location);
-                                    };
-                                },
-                                templateUrl : 'static/components-' + $stateParams.VERSION + '/models/downloadfail.tmpl.html'
-                            })
-                            .then(
-                                function (location) {
-                                    if (location) {
-                                        trainingService.deleteTrainingData(project.id, $scope.userId, vm.profile.tenant, location.imageid)
-                                            .then(function() {
-                                                $state.reload();
-                                            })
-                                            .catch(function (e) {
-                                                var errId = displayAlert('errors', e.status, e.data);
-                                                scrollToNewItem('errors' + errId);
-                                            });
-                                    }
-                                },
-                                function() {
-                                    // cancelled. do nothing
-                                }
-                            );
-                        }
-                        else {
-                            var errId = displayAlert('errors', err.status, err.data);
-                            scrollToNewItem('errors' + errId);
-                        }
-                    });
-            }
+                    refreshModels();
+
+                    $scope.submittingTrainingRequest = false;
+                })
+                .catch(function (err) {
+                    loggerService.error('[ml4kmodels] model training failed', err);
+                    $scope.submittingTrainingRequest = false;
+
+                    if (createModelFailedDueToDownloadFail(err)) {
```

Comment:
Need to update the download/resize API to return errors that this will recognize